doc/nrf/known_issues.rst

```diff
@@ -1357,90 +1387,110 @@ OCT-2501: Charging over seven hours results in error
+  **Workaround:** Use different CONFIG_BT_DEVICE_NAME for BIS and CIS.
 
   **Affected platforms:** nRF5340 Audio DK
 
+
 Controller subsystem for nRF5340 Audio
 --------------------------------------
 
 The following known issues apply to the LE Audio subsystem (NET core controller) for nRF5340 used in the nRF5340 Audio application.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
 OCX-138: Some conformance tests not passing
    Not all Bluetooth conformance tests cases pass.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
 OCX-152: OCX-146: 40 ms ACL interval may cause TWS to be unstable
   There may be combinations of ACL intervals and other controller settings that cause instabilities to connected or true wireless stereo setups.
 
   **Workaround:** Use an alternative ACL connection interval.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0
+
+OCX-153: Cannot create BIG sync after terminate pending BIG sync
+If a pending BIG sync is canceled by sending the LE Broadcast Isochronous Group Sync Estabilished
+ command, then it is impossible for the host to create a new BIG sync afterwards.
+
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
 OCX-155: Larger timestamps than intended
    The timestamps/Service Data Unit references (SDU refs), may occasionally be larger than intended and then duplicated in the next interval.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
 OCX-156: PTO is not supported
    The controller does not support Pre-Transmission Offset.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
-OCX-157: OCT-140: Interleaved broadcasts streaming issues
+OCX-157: OCX-140: Interleaved broadcasts streaming issues
   Interleaved broadcasts are unable to stream with certain Quality of Service (QoS) configurations.
 
   **Workaround:** Set retransmits (RTN) to ``<= 2`` and octets per frame to ``<= 80`` for stereo.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0
+  OCX-178: Transport latency doesn't effect flush timeout.
 
-OCX-165: Collisions of BIS and ACL on the same broadcaster/central
-   Broadcast Isochronous Stream (BIS) and Asynchronous Connection-oriented Logical transports (ACL) may collide in the scheduler.
+.. rst-class:: v2-4-0
 
-   **Workaround:** Create the ACLs before creating any BISes.
+  OCX-183: Feature request control procedure initiated when controller is in progress of
+  creating CIS and CIS is not yet established.
 
-.. rst-class:: v2-3-0 v2-2-0
+.. rst-class:: v2-4-0
 
-OCX-168: Issues with reestablishing streams
-   Syncing of broadcast receivers takes longer than in version 3310, especially for high retransmit (RTN) values.
+  OCX-184: If 0 dBm TX power is selected, the FEM/PA TX/RX pins do not toggle correctly.
+
+  **Workaround:** Set max TX power larger than 0 dBm.
+
+
+.. rst-class:: v2-4-0
+
+  OCX-188: The controller reserves some pins (0.28 - 0.31), which may collide with FEM/PA features.
```

Comment:
FEM feature cannot work properly on GPIO pins from P0.28 to P0.31. User should prevent to use these pins for FEM feature.